libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -25,77 +25,50 @@
 #include <cstdint>
 #include <limits>
 
+static const int64_t N{10000};
+
 using namespace fetch::fixed_point;
 
+using MyFPTypes = ::testing::Types<fetch::fixed_point::fp32_t, fetch::fixed_point::fp64_t,
+                                   fetch::fixed_point::fp128_t>;
+
```

Comment:
please wrap all the tests in the fetch/math/test namespace

Reply:
done